### PR TITLE
[13.0][FIX] rma: multiple substitution moves

### PR DESCRIPTION
**Pull request by chienandalu**
The product replacement could explode into several moves like in the case of MRP BoM kits.

cc @Tecnativa TT40194

please review @pedrobaeza @victoralmau

### CODE REVIEW

File: rma/models/rma.py [hunk 1]
```diff
@@ -1134,24 +1134,28 @@ def create_replace(self, scheduled_date, warehouse, product, qty, uom):
...
-                    'data-oe-model="stock.picking" data-oe-id="%d">%s</a>) '
-                    "has been created."
-                )
-                % (
-                    new_move.id,
-                    new_move.name_get()[0][1],
-                    new_move.picking_id.id,
-                    new_move.picking_id.name,
+        new_moves = self.delivery_move_ids - moves_before
+        if new_moves:
```
Comment (pedrobaeza) [inline rma/models/rma.py]: No need of this extra `if`. Just put the `for` and check if body has content.

**Comment by pedrobaeza** [comment]
Is this a real case?

**Comment by chienandalu** [comment]
> Is this a real case?

Yes. For the case that the substituted product delivery explodes in serveral moves like in mrp.bom kits

**Comment by pedrobaeza** [comment]
OK, then please document that in the code saying why there can be more than one move.

File: rma/models/rma.py (hunk 1 above)
Comment (chienandalu) [reply rma/models/rma.py]: Simplified

**Review by victoralmau — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
/ocabot merge patch

Please fw-port it.

**Pull request merged**